mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -275,6 +289,104 @@ def test_sees_all_prisons(self):
         )
 
 
+class JobInformationViewTestCase(SecurityBaseTestCase):
+    protected_views = [
+        'security:credit_list',
+        'security:dashboard',
+        'security:disbursement_list',
+        'security:prisoner_list',
+        'security:sender_list',
+    ]
+
+    @responses.activate
+    def test_redirects_when_provided_job_info_flag_is_missing(self):
+        self.login(
+            user_data=self.get_user_data(
+                flags=[
+                    confirmed_prisons_flag,
+                    hmpps_employee_flag,
+                ],
+            ),
+        )
+        for view in self.protected_views:
+            response = self.client.get(reverse(view), follow=True)
+            self.assertContains(response, '<!-- job_information -->')
+
+    @responses.activate
+    def test_can_view_app_when_user_has_provided_job_info_flag(self):
+        self.login(
+            user_data=self.get_user_data(
+                flags=[
+                    confirmed_prisons_flag,
+                    hmpps_employee_flag,
+                    provided_job_info_flag,
+                ]
+            ),
+        )
+        response = self.client.get(reverse('security:dashboard'), follow=True)
+        self.assertContains(response, '<!-- security:dashboard -->')
+
+    @responses.activate
+    def test_successfull_form_post(self):
+        mock_post_for_job_info_endpoint()
+        self.login(user_data=self.get_user_data(
+            flags=[confirmed_prisons_flag, hmpps_employee_flag])
+        )
+        responses.add(
+            responses.PUT,
+            api_url('/users/shall/flags/%s/' % provided_job_info_flag),
+            json={}
+        )
+
+        response = self.client.post(reverse('job_information'), data={
+            'job_title': 'Evidence collator',
+            'prison_estate': 'National',
+            'tasks': 'yas sir'
+        }, follow=True)
+
+        self.assertContains(response, '<!-- security:dashboard -->')
+
+    @responses.activate
+    def test_unsuccessfull_form_post(self):
+        mock_post_for_job_info_endpoint()
+        self.login(user_data=self.get_user_data(
```

Comment:
i'm slightly surprised that the mixture of a hanging indent and a normal one didn't cause flake8 to complain. i think it's best to stick to one style in a single expression